src/main/java/crazypants/enderio/conduit/gui/GuiExternalConnectionSelector.java

```diff
@@ -27,6 +29,8 @@
 
   Set<EnumFacing> cons;
   IConduitBundle cb;
+  String adjacentBlockNames[] = new String[6];
```

Comment:
Plase use Enumfacing.values().length, or even better EnumMap<EnumFacing,String> instead of the array.